flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/stream/StreamExecGlobalWindowAggregate.java

```diff
@@ -169,7 +171,11 @@ protected Transformation<RowData> translateToPlanInternal(
                 TimeWindowUtil.getShiftTimeZone(
                         windowing.getTimeAttributeType(),
                         TableConfigUtils.getLocalTimeZone(config));
-        final SliceAssigner sliceAssigner = createSliceAssigner(windowing, shiftTimeZone);
+        final WindowAssigner windowAssigner = createWindowAssigner(windowing, shiftTimeZone);
```

Comment:
What is the behavior if enable the two-phase agg forcibly with session window? Assuming the following pattern:
Local Agg -> Global Agg -> Session Window? Does it can work?

Reply:
Session window agg does not support two-stage optimization yet.